openshift/jag-csrs.yml

```diff
@@ -0,0 +1,164 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: ${APP_NAME}
+  description: Deployment template for the JAG-csrs
+  
+labels:
+  app: ${APP_NAME}
+  app.kubernetes.io/name: ${COMPONENT_NAME}
+  app.kubernetes.io/part-of: ${APP_NAME}
+
+  
+parameters:
+
+- name: APP_NAME
+  description: The application name that each component is part of
+  required: true
+  value: jag-csrs-frontend
+
+- name: SPLUNK_CONFIG_NAME
```

Comment:
This is the front end. Will the front end be logging to splunk?

Reply:
It is going to be for both front and back ends.
Just  a base or supporting work for future iterations. 
So I would leave those as is.

Reply:
Also client side logging is possible but I agree it is cumbersome. So it is our choice whether we would want to do it or not.

https://www.splunk.com/en_us/blog/tips-and-tricks/client-side-splunk.html